package-testing/sdk-test-runner/src/app.ts

```diff
@@ -115,15 +116,11 @@ export default class App {
 
     // Junit support.
     if (this.config.junitFilePath) {
-      this.writeJUnitReport(testSuiteResults, 'A report name', this.config.junitFilePath);
+      const reportName = `Eppo SDK Test: ${this.config.sdkName}`;
+      this.writeJUnitReport(testSuiteResults, reportName, this.config.junitFilePath);
     }
 
-    // Exit 1 if there were test failures
-    if (numFailures > 0) {
-      process.exit(1);
-    } else {
-      process.exit(0);
-    }
+    return numFailures > 0;
```

Comment:
it seems here we're using `true` to represent a failure whereas in other places we return `false`